python/tvm/rpc/proxy.py

```diff
@@ -512,7 +512,7 @@ def __init__(
                 self.port = my_port
                 break
             except socket.error as sock_err:
-                if sock_err.errno in [98, 48]:
+                if sock_err.errno in [98, 48, 10098, 10048]:
```

Comment:
I would recommend using symbols from the `errno` library here instead of hardcoding values.
If that does not work, could you at least add a comment here indicating what these error codes are?

Reply:
98/10098 is errno.EADDRINUSE for linux/windows. This works for both.
48 is errno.ELNRNG on linux but ELNRNG is not defined in windows so we can't use the symbol. The error translates to `Link number out of range` which I don't understand. I removed 48/10048. If we want we can add the number 48 back but we can't use the symbol.